```
src\libraries\Common\tests\System\Net\Http\ResponseStreamTest.cs(133,61): error CS0246: (NETCORE_ENGINEERING_TELEMETRY=Build) The type or namespace name 'HttpRequestOptionsKey<>' could not be found (are you missing a using directive or an assembly reference?)
```

Doesn't seem like unrelated... I assume it's because `HttpRequestOptionsKey` type doesn't exist in net48.

--- a/src/libraries/Common/tests/System/Net/Http/ResponseStreamTest.cs
+++ b/src/libraries/Common/tests/System/Net/Http/ResponseStreamTest.cs
@@ -126,41 +126,6 @@ namespace System.Net.Http.Functional.Tests
             }
         }
 
-        [OuterLoop]
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser))]
-        public async Task BrowserHttpHandler_Streaming()
-        {
-            var WebAssemblyEnableStreamingResponseKey = new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse");
-
-            var size = 1500 * 1024 * 1024;
-            var req = new HttpRequestMessage(HttpMethod.Get, Configuration.Http.RemoteSecureHttp11Server.BaseUri + "large.ashx?size=" + size);
-
-            req.Options.Set(WebAssemblyEnableStreamingResponseKey, true);
-
-            using (HttpClient client = CreateHttpClientForRemoteServer(Configuration.Http.RemoteSecureHttp11Server))
-            // we need to switch off Response buffering of default ResponseContentRead option
-            using (HttpResponseMessage response = await client.SendAsync(req, HttpCompletionOption.ResponseHeadersRead))
-            {
-                Assert.Equal(typeof(StreamContent), response.Content.GetType());
-
-                Assert.Equal("application/octet-stream", response.Content.Headers.ContentType.MediaType);
-                Assert.True(size == response.Content.Headers.ContentLength, "ContentLength");
-
-                var stream = await response.Content.ReadAsStreamAsync();
-                Assert.Equal("ReadOnlyStream", stream.GetType().Name);
-                var buffer = new byte[1024 * 1024];
-                int totalCount = 0;
-                int fetchedCount = 0;
-                do
-                {
-                    // with WebAssemblyEnableStreamingResponse option set, we will be using https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader/read
-                    fetchedCount = await stream.ReadAsync(buffer, 0, buffer.Length);
-                    totalCount += fetchedCount;
-                } while (fetchedCount != 0);
-                Assert.Equal(size, totalCount);
-            }
-        }
-
         [OuterLoop("Uses external servers", typeof(PlatformDetection), nameof(PlatformDetection.LocalEchoServerIsNotAvailable))]
         [Theory, MemberData(nameof(RemoteServersMemberData))]
         public async Task GetAsync_UseResponseHeadersReadAndCallLoadIntoBuffer_Success(Configuration.Http.RemoteServer remoteServer)
@@ -262,7 +227,43 @@ namespace System.Net.Http.Functional.Tests
                 }
             }
         }
+
 #if NETCOREAPP
+        [OuterLoop]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser))]
+        public async Task BrowserHttpHandler_Streaming()
+        {
+            var WebAssemblyEnableStreamingResponseKey = new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse");
+
+            var size = 1500 * 1024 * 1024;
+            var req = new HttpRequestMessage(HttpMethod.Get, Configuration.Http.RemoteSecureHttp11Server.BaseUri + "large.ashx?size=" + size);
+
+            req.Options.Set(WebAssemblyEnableStreamingResponseKey, true);
+
+            using (HttpClient client = CreateHttpClientForRemoteServer(Configuration.Http.RemoteSecureHttp11Server))
+            // we need to switch off Response buffering of default ResponseContentRead option
+            using (HttpResponseMessage response = await client.SendAsync(req, HttpCompletionOption.ResponseHeadersRead))
+            {
+                Assert.Equal(typeof(StreamContent), response.Content.GetType());
+
+                Assert.Equal("application/octet-stream", response.Content.Headers.ContentType.MediaType);
+                Assert.True(size == response.Content.Headers.ContentLength, "ContentLength");
+
+                var stream = await response.Content.ReadAsStreamAsync();
+                Assert.Equal("ReadOnlyStream", stream.GetType().Name);
+                var buffer = new byte[1024 * 1024];
+                int totalCount = 0;
+                int fetchedCount = 0;
+                do
+                {
+                    // with WebAssemblyEnableStreamingResponse option set, we will be using https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultReader/read
+                    fetchedCount = await stream.ReadAsync(buffer, 0, buffer.Length);
+                    totalCount += fetchedCount;
+                } while (fetchedCount != 0);
+                Assert.Equal(size, totalCount);
+            }
+        }
+
         [Theory]
         [InlineData(TransferType.ContentLength, TransferError.ContentLengthTooLarge)]
         [InlineData(TransferType.Chunked, TransferError.MissingChunkTerminator)]

